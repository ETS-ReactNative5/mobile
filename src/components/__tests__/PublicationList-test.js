import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import { PublicationList } from '../PublicationList'

const disciplines = ['science']
const publications = {
  'science': {
    'projects': {}
  }
}

it('renders correctly', () => {
  const tree = renderer.create(
    <PublicationList
      isConnected={true}
      disciplines={disciplines}
      selectedDiscipline={'science'}
      publications={publications}
      fetchPublications={jest.fn}
    />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
