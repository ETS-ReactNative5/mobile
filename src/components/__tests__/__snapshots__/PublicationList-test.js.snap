exports[`test renders correctly 1`] = `
<View
  style={undefined}>
  <ScrollView>
    <View
      style={undefined}>
      <View />
    </View>
  </ScrollView>
  <View
    style={
      Array [
        undefined,
        false,
      ]
    }>
    <View
      style={undefined}>
      <TouchableOpacity
        activeOpacity={0.7}
        onPress={[Function]}
        style={undefined}>
        <View
          style={undefined}>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                undefined,
              ]
            }>
            Choose Category
          </Text>
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontFamily": "FontAwesome",
                  "fontSize": 12,
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                undefined,
              ]
            }>
            ï„‡
          </Text>
        </View>
      </TouchableOpacity>
    </View>
  </View>
</View>
`;
